Resources/en.lproj/Localizable.strings

```diff
@@ -765,7 +765,10 @@
 "selected_profile" = "Selected profile";
 "app_profiles" = "App profiles";
 "profile_sett_descr" = "Affect selected application profile";
-"export_profile_descr" = "To import a profile, tap on the needed file (*.osf) in any application and select \"Copy to OsmAnd Maps\"";
+"export_profile_descr" = "You can export all profile settings. Copy all settings from other profile. Reset all profile settings to default state.";
```

Comment:
Why did you delete the original string? Please re-add the deleted string with a different id and use it in OAMainSettingsView controller. As for the new string, please correct it to proper English: "You can export all profile settings, copy all settings from another profile or reset all profile settings to default state."